index.html

```diff
@@ -458,6 +467,142 @@ <h4>
+              <li>Let <var data-type="object">object</var> be result of
+              <a data-cite="ecma-262#sec-json.parse">JSON-parsing</a>
+              <var>data</var>.
+              </li>
+              <li>If <var>object</var> is <code>null</code>, continue.
+              </li>
+              <li>Set <var>restrictions</var> to the result of <a data-cite=
+              "webidl#dfn-convert-ecmascript-to-idl-value">converting</a> <var>
+                object</var> into a <a>BasicCardRequest</a>.
+              </li>
+              <li>Apply <var>restrictions</var>'s <a>supportedNetworks</a>
+              member to constrain the card type inputs of the payment UI.
+              </li>
+              <li>Break.
+              </li>
+            </ol>
+          </li>
+          <li>If an payment instrument is selected by default, <a>apply the
+          modifiers</a> using <var>request</var><a data-cite=
+          "payment-request/#dfn-serializedmodifierdata">[[\serializedModifierData]]</a>
+          and the <a>network</a> of the selected instrument.
+          </li>
+        </ol>
+      </section>
+      <section>
+        <h2>
+          Applying the modifiers
+        </h2>
+        <p>
+          The steps to <dfn>apply the modifiers</dfn> are given by the
+          following algorithm. It takes a <a>PaymentRequest</a> <var data-type=
+          "PaymentRequest">request</var> and a <a>network</a>
+          <var>networkIdentifier</var> :
+        </p>
+        <ol class="algorithm">
+          <li>Let <var>index</var> the length of
+          <var>request</var>.<a data-cite=
+          "payment-request/#dfn-serializedmodifierdata">[[\serializedModifierData]]</a>
+          </li>
+          <li>For each <a data-cite="infra#pair">pair</a> (<a>DOMString</a>
+          <var>id</var> / string <var>data</var>) of
+          <var>request</var>.<a data-cite=
+          "payment-request/#dfn-serializedmodifierdata">[[\serializedModifierData]]</a>
+          in reverse order:
+            <ol>
+              <li>Decrement <var>index</var> by 1.
+              </li>
+              <li>If <var>id</var> is not "<a>basic-card</a>", continue.
+              </li>
+              <li>Let <var>object</var> be result of <a data-cite=
+              "ecma-262#sec-json.parse">JSON-parsing</a> <var>data</var>.
+              </li>
+              <li>If <var>object</var> is <code>null</code>, continue.
+              </li>
+              <li>Let <var>bcRequest</var> be the result of converting
+              <var>object</var> to <a>BasicCardRequest</a>.
+              </li>
+              <li>If <var>bcRequest</var>'s <a>supportedNetworks</a> is zero
+              length, or <a>supportedNetworks</a> includes
+              <var>networkIdentifier</var>:
+                <ol>
+                  <li>Let <var>modifier</var> be the PaymentMethodModifier from
+                  <var>request</var>.[[\details].modifiers at <var>index</var>.
```

Comment:
```suggestion
                  <var>request</var>.[[details]].modifiers at <var>index</var>.
```

Reply:
Tgis one is correct. The “\” implies “internal slot” rather than citation.

Reply:
Hmm elsewhere I see `request.[[serializedMethodData]]` being used to reference the `[[serializedMethodData]]` internal slot of `request`. Perhaps I'm missing something.

Reply:
Yes, you are absolutely correct. I need to go through and fix all those and a bunch of links. Just wanted to get this up for a sanity check. Will fix it up tomorrow.

Reply:
SG. Thanks!